renderdoc/CMakeLists.txt

```diff
@@ -0,0 +1,267 @@
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+if(CMAKE_COMPILER_IS_GNUCXX)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden -fvisibility-inlines-hidden")
+endif()
+
+set(RDOC_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(RDOC_DEFINITIONS PRIVATE -DRENDERDOC_EXPORTS)
+set(RDOC_INCLUDES
+    PRIVATE ${RDOC_SOURCE_DIR}
+    PRIVATE ${RDOC_SOURCE_DIR}/3rdparty)
+set(RDOC_LIBRARIES)
+
+# get git commit hash
+execute_process(COMMAND sh ${CMAKE_SOURCE_DIR}/calc_hash.sh
+    OUTPUT_VARIABLE GIT_COMMIT_HASH)
+string(STRIP ${GIT_COMMIT_HASH} GIT_COMMIT_HASH)
+list(APPEND RDOC_DEFINITIONS
+    PRIVATE -DGIT_COMMIT_HASH="${GIT_COMMIT_HASH}")
+
+if(WIN32)
```

Comment:
Harmless, but there's no need for this if() case unless there will be other platforms in future (I don't know if android will need a different branch here, if it does then ignore this comment).